### ISSUE TITLE
Refine line number runs

### ISSUE THREAD

**Issue by rgbkrk** [rgbkrk/atom-script#426]
A lot of people seem a bit confused by the statement about runs by line number. It's almost like people want a "run at line" option. What do you think @erran?

**Comment by erran** [comment]
You're right. We could come up with a new name for line number based runs *or* we could rename it to "run line" which will default to running it as a selection. Thoughts?
